### PR TITLE
Stitch creditCard on Exchange Order

### PR DESCRIPTION
**Pull request by ashkan18**
# Problem
When we stitched Exchange schema we missed resolving `Order.creditCardId` and adding `Order.creditCard`.

# Solution 
Stitch `creditCard` using `Order.creditCardId` on both `buy` and `offer` orders.

# Problem
When we try to `delegateToSchema` the `fieldName` on V1 is `credit_card` while this has change in V2 to `creditCard`. Current stitched code is shared between V1 and V2 and current code would only work for V1.
Knowing this is only used by Reaction (`V1`) at the moment, this may be fine for now but we may want to duplicate exchange stitching to get `V2` version actually working.

### CODE REVIEW

File: _schemaV2.graphql [hunk 1]
```diff
@@ -4474,46 +4477,6 @@ type Gene implements Node {
     last: Int
   ): ArtistConnection
   artworksConnection(
-    acquireable: Boolean
```
Comment (ashkan18) [inline _schemaV2.graphql]: not sure why these are here! seem pretty unrelated to my changes!

File: src/lib/stitching/exchange/stitching.ts [hunk 2]
```diff
@@ -105,6 +105,24 @@ export const exchangeStitchingEnvironment = (
...
   })
 
+  const creditCardResolver = {
+    fragment: `fragment CommerceOrderCreditCard on CommerceOrder { creditCardId }`,
+    resolve: (parent, _args, context, info) => {
+      const id = parent.creditCardId
+      return info.mergeInfo.delegateToSchema({
+        schema: localSchema,
+        operation: "query",
+        fieldName: "credit_card",
```
Comment (ashkan18) [inline src/lib/stitching/exchange/stitching.ts]: for this to work for `V2` this has to change to `creditCard`

**Comment by alloy** [comment]
> # Problem
> When we try to `delegateToSchema` the `fieldName` on V1 is `credit_card` while this has change in V2 to `creditCard`. Current stitched code is shared between V1 and V2 and current code would only work for V1.
> Knowing this is only used by Reaction (`V1`) at the moment, this may be fine for now but we may want to duplicate exchange stitching to get `V2` version actually working.

https://artsyproduct.atlassian.net/browse/PLATFORM-1672

**Comment by alloy** [comment]
LGTM 👍
